tests/flag_set.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+ * This file is part of liblcf. Copyright (c) 2018 liblcf authors.
+ * https://github.com/EasyRPG/liblcf - https://easyrpg.org
+ *
+ * liblcf is Free/Libre Open Source Software, released under the MIT License.
+ * For the full copyright and license information, please view the COPYING
+ * file that was distributed with this source code.
+ */
+
```

Comment:
Please add this file in `Makefile.am` around line 220.

Reply:
This has been added